ci-cd/Setup.md

```diff
@@ -7,15 +6,34 @@
 - Jenkins
 - TOMCAT
 
-### Machine Setup Instructions:
+# Machine Setup Instructions:
+
+### OS: Windows 7+
+### JDK Setup
+- [JDK 8 Download](http://www.oracle.com/technetwork/pt/java/javase/downloads/jdk8-downloads-2133151.html?printOnly=1)
+    - 64 bit OS : jdk-8u171-windows-x64.exe
+    - 32 bit OS : jdk-8u171-windows-i586.exe
+
+### Maven Setup
+- [Maven Download](http://apache.mirrors.pair.com/maven/binaries/)
+    - apache-maven-3.2.2-bin.zip 
+- [Install Instructions](https://maven.apache.org/install.html)
+
+### IDE Setup [ IntelliJ Community Edition ]
+- [Community Edition Download](https://www.jetbrains.com/idea/download/#section=windows)
+
+### Git Setup [ Command Line ]
+- [Git for Windows Download](https://git-scm.com/download/win)
+
+### Jenkins Setup [ Continuous Integration Build Tool ]
+- [Jenkins Download & Install Instructions](https://jenkins.io/doc/pipeline/tour/getting-started/)
+
+### TOMCAT Setup
```

Comment:
We should be using Embedded Tomcat to avoid this to them

Reply:
Agree I changed TOMCAT to SpringBoot 2.0.2 GA Release